src/interfaces/IQuoter.sol

```diff
@@ -74,26 +56,20 @@ interface IQuoter {
     /// exactAmount The desired output amount
     /// sqrtPriceLimitX96 The price limit of the pool that cannot be exceeded by the swap
     /// hookData arbitrary hookData to pass into the associated hooks
-    /// @return deltaAmounts Delta amounts resulted from the swap
-    /// @return sqrtPriceX96After The sqrt price of the pool after the swap
-    /// @return initializedTicksLoaded The number of initialized ticks that the swap loaded
+    /// @return amountIn The input quote for the exactOut swap
+    /// @return gasEstimate Estimated gas units used for the swap
     function quoteExactOutputSingle(QuoteExactSingleParams memory params)
         external
-        returns (int128[] memory deltaAmounts, uint160 sqrtPriceX96After, uint32 initializedTicksLoaded);
+        returns (uint256 amountIn, uint256 gasEstimate);
 
     /// @notice Returns the delta amounts along the swap path for a given exact output swap
     /// @param params the params for the quote, encoded as 'QuoteExactParams'
     /// currencyOut The output currency of the swap
     /// path The path of the swap encoded as PathKeys that contains currency, fee, tickSpacing, and hook info
     /// exactAmount The desired output amount
-    /// @return deltaAmounts Delta amounts along the path resulted from the swap
-    /// @return sqrtPriceX96AfterList List of the sqrt price after the swap for each pool in the path
-    /// @return initializedTicksLoadedList List of the initialized ticks that the swap loaded for each pool in the path
+    /// @return amountIn The input quote for the exactOut swap
+    /// @return gasEstimate Estimated gas units used for the swap
     function quoteExactOutput(QuoteExactParams memory params)
         external
-        returns (
-            int128[] memory deltaAmounts,
-            uint160[] memory sqrtPriceX96AfterList,
-            uint32[] memory initializedTicksLoadedList
-        );
+        returns (uint256 amountIn, uint256 gasEstimate);
```

Comment:
Ok I think we should actually return the BalanceDelta, will follow up async w you